LICENSES/headers/MIT-Clause.txt

```diff
@@ -0,0 +1,22 @@
+-*- coding: utf-8 -*-
+Copyright IRT Antoine de Saint Exupéry et Université Paul Sabatier Toulouse III - All
```

Comment:
It has to be checked but for me you cannot be "All rights reserved" and MIT licence. The copyright is legit though